app/forms/Countries.scala

```diff
@@ -31,13 +31,28 @@ object Countries {
       .map(_.replace("\"", ""))
       .toList
 
+  val countriesThreeDigitMap : Map[String, String] = {
+    val filename = "conf/three-digit-country-code-to-country-name.csv"
```

Comment:
I'm not sure this file should be in the conf file folder. seems more apt to be in the resources package

Reply:
resources has not been used in any of our repos